addon-test-support/@ember/test-helpers/dom/-logging.ts

```diff
@@ -0,0 +1,65 @@
+import Target from './-target';
+
+/**
+ * Logs a debug message to the console if the `testHelperLogging` query
+ * parameter is set.
+ *
+ * @private
+ * @param {string} helperName Name of the helper
+ * @param {string|Element} target The target element or selector
+ */
+export function log(helperName: string, target: Target, ...args: any[]) {
+  if (loggingEnabled()) {
+    // eslint-disable-next-line no-console
+    console.log(`${helperName}(${[elementToString(target), ...args.filter(Boolean)].join(', ')})`);
+  }
+}
+
+/**
+ * Returns whether the test helper logging is enabled or not via the
+ * `testHelperLogging` query parameter.
+ *
+ * @private
+ * @returns {boolean} true if enabled
+ */
+function loggingEnabled(): boolean {
+  return typeof location !== 'undefined' && location.search.indexOf('testHelperLogging') !== -1;
+}
+
+/**
+ * This generates a human-readable description to a DOM element.
+ *
+ * @private
+ * @param {*} el The element that should be described
+ * @returns {string} A human-readable description
+ */
+function elementToString(el: unknown) {
```

Comment:
Can you add some tests for this function? Doesn't need to be 100% exaustive, just something that confirms the output is reasonable...

Reply:
Should this short circuit if `el` is a string? I think it would eventually do that (due to the condition of `!HTMLElement && !SVGElement`), but IMHO it would be easier to parse if the most common case was out of the way first...

Reply:
yeah, I can probably just copy the tests from qunit-dom over here :)

Reply:
done ✅